integrations/batcher_test.go

```diff
@@ -2605,6 +2602,42 @@ data
 	testBatcherWithOutput(t, "TestBatch_StateTracking", script, 8*time.Second, er, false)
 }
 
+func TestBatch_Trickle(t *testing.T) {
+	var script = `
+	var data = batch
+	|query('SELECT value FROM "telegraf"."default"."cpu"')
+		.period(4s)
+		.every(4s)
+		.groupBy('host')
+data
+	|trickle()
+	|window().period(10s)
+	|httpOut('TestBatch_Trickle')`
```

Comment:
Checking my understanding: this test is showing that `trickle()` followed by `window` is a way to re-batch data? i.e. in this case, re-batching from 4s windows into 10s windows?